meta_automl/data_preparation/pipeline_features_extractors/fedot_pipeline_features_extractor.py

```diff
@@ -0,0 +1,145 @@
+import json
+from typing import Any, Dict, List, Optional, Union
+
+import numpy as np
+import torch
+from torch_geometric.data.data import Data
+
+
+class FEDOTPipelineFeaturesExtractor:
+    """FEDOT pipeline features extractor.
+
+    List of extracted features: directed adjacency matrix, nodes operation type.
+
+    Parameters:
+    -----------
+    include_operations_hyperparameters: Whether to include operation hyperparameters. Currently not supported. Default: `False`.
+    operation_type2name: Mapping of operation type to its implementation name. Default mapping is adapted from FEDOT.
+    operation_encoding: Type of operation encoding. Can be `"ordinal"` or `"onehot"`. Default: `"ordinal"`.
+    """
+
+    default_operation_type2name = {
+        # '': '',  # Empty string indicates that the node means <no_operation> as https://t.me/FGksjp67 requested.
```

Comment:
"Замечательный" и ненужный комментарий 😠

Reply:
Ну да, с учетом что его по какой-то причине не удалось сделать, если ничего не изменилось.
Заменить на: 
```suggestion
        # '': '',  # Empty string indicates that the node means <no_operation> 
```
?